src/js/src/isBrowser.js

```diff
@@ -0,0 +1,10 @@
+// @flow strict
+
+// https://stackoverflow.com/questions/17575790/environment-detection-node-js-or-browser/31090240#31090240
+// eslint-disable-next-line no-new-func
+const testIsBrowser = new Function('try {return this===window;}catch(e){ return false;}');
+
+export default function isBrowser(): boolean {
+  // We need to add !! to be able to type the return type as boolean
+  return !!testIsBrowser();
```

Comment:
`testIsBrowser() === true` not possible?

Reply:
Probably yes, is it somehow better?

Reply:
probably not, comment should not be needed in that case :)

Reply:
I feel it would be necessary anyway. I need to explain why we simply don't do `return testIsBrowser()`, and that doesn't change with `return testIsBrowser() === true`